main/guides/getting-started/how-to-use-ui-kit.md

```diff
@@ -0,0 +1,142 @@
+# Introduction to Agoric UI Kit
+The Agoric UI Kit is a collection of components and tools designed to help developers build graphical user interfaces for Agoric decentralized applications (dapps). It provides a set of reusable React components, utilities for reading contract data from vstorage, functions for connecting to the user's wallet, and methods for executing offers on the Agoric blockchain.
+
+## Setup
+To use the Agoric UI Kit, you need to set up SES in your application environment. SES is a runtime for JavaScript that ensures the security and integrity of your Agoric dapp. Refer to the [SES setup example](https://github.com/Agoric/dapp-inter/blob/main/src/main.tsx#L1) for instructions on enabling SES in your application. 
+
+## React Components
+The UI Kit provides a set of React components in the `packages/react-components` directory. These components are designed to make it easier to build user interfaces for Agoric dapps using React. To use these components, simply import them into your React application and integrate them into your UI.
```

Comment:
This phrase doesn't explain much, we should eventually have a separate page/section based on https://github.com/Agoric/ui-kit/blob/main/packages/react-components/README.md

Reply:
add a link to there in the mean time?

eventually we should be generating ui-kit reference docs
 - https://github.com/Agoric/documentation/issues/1032

but for now, some link seems better than nothing

Reply:
Oh yea I'd love to do something like that. Ideally we could have a dropdown menu that allows you to switch package versions as well.

Reply:
Removed the intro sentences and replaced with link to ui-kit README.md file.